src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -3504,6 +3505,54 @@ private BoundExpression BindNullCoalescingOperator(BinaryExpressionSyntax node,
             return GenerateNullCoalescingBadBinaryOpsError(node, leftOperand, rightOperand, Conversion.NoConversion, diagnostics);
         }
 
+        private BoundExpression BindNullCoalescingAssignmentOperator(AssignmentExpressionSyntax node, DiagnosticBag diagnostics)
```

Comment:
Nit: consider adding an xml doc on this method, with an example syntax: `expr ??= expr2`

Reply:
Will address in a follow up.

---
In reply to: [201188415](https://github.com/dotnet/roslyn/pull/28253#discussion_r201188415) [](ancestors = 201188415)